consumers/writers/influxdb/fields.go

```diff
@@ -9,7 +9,7 @@ import (
 type fields map[string]interface{}
 
 func senmlFields(msg senml.Message) fields {
-	updateTime := strconv.FormatFloat(msg.UpdateTime, 'f', -1, 64)
+	// updateTime := strconv.FormatFloat(msg.UpdateTime, 'f', -1, 64)
```

Comment:
You can simply remove it

Reply:
Okay, but I have checked that the build fails, I thing i will need to remove the strconv package also and replace retur
n value of updateTime with msg.UpdateTime